ui/tests/helpers/resolver.js

```diff
@@ -8,4 +8,6 @@ resolver.namespace = {
   podModulePrefix: config.podModulePrefix,
 };
 
+resolver.pluralizedTypes.ability = 'abilities';
```

Comment:
This was necessary?

Reply:
hah WELL I feel like it must have been at some point but I’ve just removed it with no resulting error so apparently not now… thanks for the prompt 🙃